src/main/java/lotto/controller/LottoController.java

```diff
@@ -8,23 +8,25 @@
 import lotto.dto.TicketDto;
 import lotto.view.LottoView;
 
+import static lotto.domain.Prize.SECOND;
+
 public class LottoController {
-    private final LottoTicketSeller seller;
     private final LottoView view;
 
     public LottoController() {
-        this.seller = new LottoTicketSeller();
         this.view = new LottoView();
     }
 
     public void start() {
         LottoPurchaseBudget budget = getBudget();
-        List<LottoTicket> tickets = seller.generateTickets(budget);
+        int manualLottoCount = getManualLottoCount(budget);
 
-        view.printTickets(mapToTicketDto(tickets));
+        LottoTickets tickets = getManualLottoTickets(manualLottoCount, budget);
+        tickets.addAll(LottoTicketSeller.purchaseAutoLottoTickets(budget));
```

Comment:
아래와 같이 LottoTickets.`addAll`의 결과로 새로운 `LottoTickets` 객체를 반환하면 어떨까요? 🙂
```suggestion
        LottoTickets manualLottoTickets = getManualLottoTickets(manualLottoCount, budget);
        LottoTickets autoLottoTickets = LottoTicketSeller.purchaseAutoLottoTickets(budget);
        LottoTickets lottoTickets = manualLottoTickets.addAll(autoLottoTickets);
```